repair/row_level.cc

```diff
@@ -662,18 +662,30 @@ future<std::list<repair_row>> to_repair_rows_list(repair_rows_on_wire rows, sche
     co_return std::move(row_list);
 }
 
-void flush_rows(schema_ptr s, std::list<repair_row>& rows, lw_shared_ptr<repair_writer>& writer) {
+void flush_rows(schema_ptr s, std::list<repair_row>& rows, lw_shared_ptr<repair_writer>& writer, locator::effective_replication_map_ptr erm, bool small_table_optimization) {
     auto cmp = position_in_partition::tri_compare(*s);
     lw_shared_ptr<mutation_fragment> last_mf;
     lw_shared_ptr<const decorated_key_with_hash> last_dk;
+    bool enable_opt = erm && small_table_optimization;
```

Comment:
If "small table optimization" is a vague and undocumented flag, "enable_opt" is even worse. You could have just called this variable "b" :-(
In fact, only after writing this comment I realized that "enable_opt" doesn't mean "optionally enable" (enable what?) but a shortcut of "enable optimization", or really "enable small table optimization". I didn't guess this on the first (or second) read.

Reply:
The enable_opt is just a local variable which I want to make it shorter. I can make it longer if this really bothers you.